package.json

```diff
@@ -31,6 +31,14 @@
   "devDependencies": {
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.6",
-    "jest-enzyme": "^7.1.2"
+    "jest-enzyme": "^7.1.2",
+    "jest-junit": "^12.0.0",
+    "junitxml-to-javascript": "^1.1.4",
+    "node-fetch": "^2.6.1"
+  },
+  "jest-junit": {
+    "suiteNameTemplate": "{filename}",
+    "classNameTemplate": "{filename}",
```

Comment:
This configuration tells the junit reporter to consider each test file as a distinct `<testsuite>`